stack.yaml

```diff
@@ -0,0 +1,6 @@
+resolver: lts-11.16
```

Comment:
This is a rather arbitrary LTS version at this point. Maybe do the thing where you include multiple stack.yamls (for this one, whatever the current one is and maybe "latest" however you do that), and build them all in travis. Not sure if it's really that simple, but I've seen it done.

Reply:
What I did was override the resolver in travis.yml to also build current LTS and the nightly one.